code/src/components/CustomCheckbox.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+export const CustomCheckbox = ({ isChecked, onChangeHandler }) => {
+  const [checked, setChecked] = useState(false);
+
+  const handleOnChange = event => {
+    onChangeHandler();
+    setChecked(event.target.checked);
+  };
+
+  const Checkbox = ({ className, checked, ...props }) => (
+    <CheckboxContainer>
+      <HiddenCheckbox checked={checked} {...props}></HiddenCheckbox>
+      <StyledCheckbox checked={isChecked}>
+        <Icon viewBox="0 0 24 24">
+          <polyline points="20 6 9 17 4 12" />
+        </Icon>
```

Comment:
The Icon component is declared as an SVG component on line 45 `const Icon = styled.svg`.
The original checkbox is hidden (rendered off screen), but as it is still present it will continue to be picked up by a screen reader.